docs/commands.md

```diff
@@ -0,0 +1,96 @@
+<!--[metadata]>
++++
+title = "Understand the typical commands"
+description = "Description of key notary commands"
+keywords = ["docker, notary, notary-client"]
++++
+<![end-metadata]-->
+
+# Understand the typical notary commands
+
+## Terminology
+
+1. GUN: Notary uses Globally Unique Names (GUNs) to identify trust collections
```

Comment:
That would probably be good to have a brief overview of the roles, and then maybe link to the TUF spec for more info?